sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -53,6 +53,17 @@ class AutoLockRenew(object):
     """
 
     def __init__(self, executor=None, max_workers=None):
+        # type: (ThreadPoolExecutor, int) -> None
+        """Auto renew locks for messages and sessions using a background thread pool.
+
+        :param executor: A user-specified thread pool. This cannot be combined with
+         setting `max_workers`.
+        :type executor: ~concurrent.futures.ThreadPoolExecutor
+        :param max_workers: Specify the maximum workers in the thread pool. If not
+         specified the number used will be derived from the core count of the environment.
+         This cannot be combined with `executor`.
+        :type max_workers: int
+        """
```

Comment:
we already have the docstring under "class AutoLockRenew".
I don't think we need duplicate for the `__init__` method.

Reply:
This is technically in our python guidelines.

I had the same eyebrow-raise as you though, so this was _also_ in the list of confirmations I asked Anna for last week; should probably be so rude as to ping on that again to unblock this. @annatisch

Reply:
Echoing our email discussion, see [here](https://review.docs.microsoft.com/en-us/help/onboard/admin/reference/python/documenting-api?branch=master#constructor-parameters)